src/azure-cli/azure/cli/command_modules/servicefabric/template/linux/template.json

```diff
@@ -462,7 +459,10 @@
             "properties": {
                 "overprovision": "[variables('overProvision')]",
                 "upgradePolicy": {
-                    "mode": "Automatic"
+                    "mode": "Automatic",
+                    "automaticOSUpgradePolicy": {
+                        "enableAutomaticOSUpgrade":  true
+                    }
                 },
                 "virtualMachineProfile": {
                     "extensionProfile": {
```

Comment:
can you update the linux extension version to
"typeHandlerVersion": "2.0",
and add
"enableAutomaticUpgrade": true,

as explained here: https://eng.ms/docs/products/azure-service-fabric/release-notes/sf-extension/sfrp-linux-extension
If you want, you can add it in a separate PR but we should change it because we are going to deprecate versions < 2.0

Reply:
updated